### DISCUSSION TITLE
Grid System?

### DISCUSSION THREAD

**Discussion by hamedfaryabi** [tailwindlabs/tailwindcss#2597] [Help] [answered]
Does tailwind css have a grid system like bootstrap that can be used in UI design?
Ineed to know `number of columns` and `gutter width`

Like this link: https://getbootstrap.com/docs/4.0/layout/grid/#grid-options

**Comment by adamwathan** [comment, marked as answer]
Yep Tailwind has all kinds of grid utilities based on CSS Grid Lauout. Start here:

https://tailwindcss.com/docs/grid-template-columns#usage
